### ISSUE TITLE
UnsupportedOperation: fileno

### ISSUE THREAD

**Issue by mateusb12** [explosion/spaCy#11236]
## How to reproduce the behaviour
When I use IDEs, spacy package works like a charm.
But, whenever I try to run spacy on a jupyter environment I get this error

![spacy](https://user-images.githubusercontent.com/19989782/181394376-958a78b0-6e60-4eb8-9c1f-8b4ecb2923a3.png)

I tried several things, including reinstalling python, and switching between venv and anaconda. Nothing worked

I also tried to search about that error on internet, but nothing was related with spacy package itself

## Your Environment

* Operating System: Windows10
* Python Version Used: 3.9.12
* spaCy Version Used: 3.4.1
* Environment Information: 

![image](https://user-images.githubusercontent.com/19989782/181394521-5248b072-8fda-49b8-9754-396bc8c20117.png)

**Comment by polm** [comment]
Thanks for the report, we'll take a look at this.

Also, please copy and paste code/terminal output as text, not as images.

**Comment by gsbelarus** [comment]
The same error. Spacy just stopped working inside notebooks. Something is broken in recent release.

**Comment by polm** [comment]
This seems to be an issue with the latest release of wasabi. We've already yanked the release on PyPI and are working on a fix, but if you're affected by this you can downgrade wasabi with `pip install wasabi==0.9.1`. 

Note this should only affect Windows users who installed spaCy in roughly the last day.

**Comment by thelunararmy** [comment]
Same problem. first I validated my install:

`! python -m spacy validate` 

Result:

```

| Loading compatibility table...
[+] Loaded compatibility table

================= Installed pipeline packages (spaCy v3.4.1) =================
[i] spaCy installation:
C:\Users\<redacted>\AppData\Roaming\Python\Python39\site-packages\spacy

NAME             SPACY            VERSION      
en_core_web_md   >=3.4.0,<3.5.0   3.4.0     [+]
```

Works perfectly fine on CLI or in the interactive interpreter:
```
Python 3.9.12 (main, Apr  4 2022, 05:22:27) [MSC v.1916 64 bit (AMD64)] :: Anaconda, Inc. on win32

Warning:
This Python interpreter is in a conda environment, but the environment has
not been activated.  Libraries may fail to load.  To activate this environment
please see https://conda.io/activation

Type "help", "copyright", "credits" or "license" for more information.
>>> import spacy
>>> help(spacy)
Help on package spacy:

NAME
    spacy

PACKAGE CONTENTS
    __main__
    about
    attrs
    cli (package)
```

But running inside Jupyter Notebooks yields a `UnsupportedOperation` Exception:
```python
import spacy
# nlp = spacy.load("en_core_web_md")
```

Err out:
```py
UnsupportedOperation                      Traceback (most recent call last)
Input In [1], in <cell line: 1>()
----> 1 import spacy

File ~\AppData\Roaming\Python\Python39\site-packages\spacy\__init__.py:6, in <module>
      3 import sys
      5 # set library-specific custom warning handling before doing anything else
----> 6 from .errors import setup_default_warnings
      8 setup_default_warnings()  # noqa: E402
     10 # These are imported as part of the API

File ~\AppData\Roaming\Python\Python39\site-packages\spacy\errors.py:2, in <module>
      1 import warnings
----> 2 from .compat import Literal
      5 class ErrorsWithCodes(type):
      6     def __getattribute__(self, code):

File ~\AppData\Roaming\Python\Python39\site-packages\spacy\compat.py:3, in <module>
      1 """Helpers for Python and platform compatibility."""
      2 import sys
----> 3 from thinc.util import copy_array
      5 try:
      6     import cPickle as pickle

File ~\AppData\Roaming\Python\Python39\site-packages\thinc\__init__.py:5, in <module>
      2 import numpy
      4 from .about import __version__
----> 5 from .config import registry

File ~\AppData\Roaming\Python\Python39\site-packages\thinc\config.py:13, in <module>
     11 from pydantic.main import ModelMetaclass
     12 from pydantic.fields import ModelField
---> 13 from wasabi import table
     14 import srsly
     15 import catalogue

File ~\AppData\Roaming\Python\Python39\site-packages\wasabi\__init__.py:12, in <module>
      9 from .util import MESSAGES  # noqa
     10 from .about import __version__  # noqa
---> 12 msg = Printer()

File ~\AppData\Roaming\Python\Python39\site-packages\wasabi\printer.py:56, in Printer.__init__(self, pretty, no_print, colors, icons, line_max, animation, animation_ascii, hide_animation, ignore_warnings, env_prefix, timestamp)
     54 self.pretty = pretty and not env_no_pretty
     55 self.no_print = no_print
---> 56 self.show_color = supports_ansi() and not env_log_friendly
     57 self.hide_animation = hide_animation or env_log_friendly
     58 self.ignore_warnings = ignore_warnings

File ~\AppData\Roaming\Python\Python39\site-packages\wasabi\util.py:264, in supports_ansi()
    262     if "ANSICON" in os.environ:
    263         return True
--> 264     return _windows_console_supports_ansi()
    266 return True

File ~\AppData\Roaming\Python\Python39\site-packages\wasabi\util.py:236, in _windows_console_supports_ansi()
    233     if not ok:
    234         raise ctypes.WinError()
--> 236 console = msvcrt.get_osfhandle(sys.stdout.fileno())
    237 try:
    238     # Try to enable ANSI output support
    239     flags = GetConsoleMode(console)

File ~\anaconda3\lib\site-packages\ipykernel\iostream.py:310, in OutStream.fileno(self)
    308     return self._original_stdstream_copy
    309 else:
--> 310     raise io.UnsupportedOperation("fileno")

UnsupportedOperation: fileno
```

## Platform Info:

- Windows 11 Pro 64-bit (10.0, Build 22000)
- Jupyter Lab 3.3.2 
- Python 3.9.12
- spaCy v3.4.1
- en_core_web_md 3.4.0

**Comment by thelunararmy** [comment]
> This seems to be an issue with the latest release of wasabi. We've already yanked the release on PyPI and are working on a fix, but if you're affected by this you can downgrade wasabi with `pip install wasabi==0.9.1`.
> 
> Note this should only affect Windows users who installed spaCy in roughly the last day.

Fixed it for me. For Jupyter users, run the following command in a cell:
```
! pip uninstall wasabi -y
```
Then:
```
! pip install wasabi==0.9.1
```
Restart the kernel. 

Thanks @polm

**Comment by adrianeboyd** [comment]
Thanks for the report, this should be resolved by the new release for wasabi v0.10.1.
